AWSTranscribeStreaming/Internal/AWSSRWebSocketDelegateAdaptor.m

```diff
@@ -174,9 +175,42 @@ - (void)webSocket:(AWSSRWebSocket *)webSocket
     });
 }
 
+#pragma mark - Optional protocol methods to fulfill Socket Rocket protocol
```

Comment:
these methods have to stay unfortunately for socket rocket to work.

Reply:
I believe delegate in the new protocol you have created does make sense to be optional. If the caller does not set the delegate then it's basically saying that they do not care about if the socket was connected, disconnected, if a message was received on the socket.

Reply:
It's not "unfortunate" so much as "the point of this class" :) I think you can go back to the previous implementation, in which these delegate methods are invoking the client delegate methods?